src/java/org/apache/cassandra/tools/StandaloneScrubber.java

```diff
@@ -112,56 +112,58 @@ public static void main(String args[])
+
+                        @Override
+                        public void run() {
+                            try {
+                                Scrubber scrubber = new Scrubber(cfs, sstable, options.skipCorrupted,
+                                        !options.noValidate, handler, true);
+                                try {
+                                    scrubber.scrub();
+                                } finally {
+                                    scrubber.close();
+                                }
+
+                                if (manifest != null) {
+                                    if (scrubber.getNewInOrderSSTable() != null)
+                                        manifest.add(scrubber.getNewInOrderSSTable());
+
+                                    List<SSTableReader> added = scrubber.getNewSSTable() == null
+                                            ? Collections.<SSTableReader> emptyList()
+                                            : Collections.singletonList(scrubber.getNewSSTable());
+                                    manifest.replace(Collections.singletonList(sstable), added);
+                                }
+
+                                // Remove the sstable (it's been copied by scrub and snapshotted)
+                                sstable.markObsolete();
+                                sstable.releaseReference();
+                            } catch (Exception e) {
+                                System.err.println(String.format("Error scrubbing %s: %s", sstable, e.getMessage()));
+                                e.printStackTrace(System.err);
+                            }
                         }
-                        finally
-                        {
-                            scrubber.close();
-                        }
-
-                        if (manifest != null)
-                        {
-                            if (scrubber.getNewInOrderSSTable() != null)
-                                manifest.add(scrubber.getNewInOrderSSTable());
-
-                            List<SSTableReader> added = scrubber.getNewSSTable() == null
-                                ? Collections.<SSTableReader>emptyList()
-                                : Collections.singletonList(scrubber.getNewSSTable());
-                            manifest.replace(Collections.singletonList(sstable), added);
-                        }
-
-                        // Remove the sstable (it's been copied by scrub and snapshotted)
-                        sstable.markObsolete();
-                        sstable.releaseReference();
-                    }
-                    catch (Exception e)
-                    {
-                        System.err.println(String.format("Error scrubbing %s: %s", sstable, e.getMessage()));
-                        e.printStackTrace(System.err);
-                    }
+                    }));
+                }
+                executor.shutdown();
```

Comment:
Nit: could awaitTermination be useful here, i guess you are already waiting on the futures to finish so it may not be worth it.